charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,289 @@
+            - 'metric.name == "system.paging.operations" and attributes["type"] == "cached"'
+      filter/exclude_network:
+        metrics:
+          datapoint:
+            - 'IsMatch(metric.name, "^system.network.*") == true and attributes["device"] == "lo"'
+
+      attributes/exclude_system_paging:
+        include:
+          match_type: strict
+          metric_names:
+            - system.paging.operations
+        actions:
+          - key: type
+            action: delete
+
+      resource:
+        attributes:
+          - key: host.id
+            from_attribute: host.name
+            action: upsert
+          # TODO chris: Upsert only when cluster name not found (resource detection override: true)
+          - key: k8s.cluster.name
+            action: upsert
+            value: {{ include "newrelic.common.cluster" . }}
+          - key: newrelicOnly
+            action: upsert
+            value: 'true'
+          - key: service.name
+            action: delete
+          - key: service_name
+            action: delete
+
+      k8sattributes:
+        auth_type: "serviceAccount"
+        passthrough: false
+        filter:
+          node_from_env_var: KUBE_NODE_NAME
+        extract:
+          metadata:
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.deployment.name
+            - k8s.namespace.name
+            - k8s.node.name
+            - k8s.pod.start_time
+        pod_association:
+          - sources: 
+            - from: resource_attribute
+              name: k8s.pod.uid
+
+      batch:
+
+    exporters:
+      otlphttp/newrelic:
+        endpoint: {{ .Values.newRelicEndpoint }}
+        headers:
+          api-key: ${env:NR_LICENSE_KEY}
+      # uncomment when writing daemonset metrics through deployment
+      # otlp:
+      #   endpoint: "nr-k8s-otel-collector-gateway:4317"
+      #   tls:
+      #     insecure: true
+      logging:
+        verbosity: detailed
+
+    service:
+      telemetry:
+        logs:
+          level: "debug"
+      pipelines:
+        metrics:
+          receivers: [otlp, hostmetrics, kubeletstats, prometheus]
+          processors:
+            # - transform/truncate
+            - filter/exclude_cpu_utilization
+            - filter/exclude_memory_utilization
+            - filter/exclude_memory_usage
+            - filter/exclude_filesystem_utilization
+            - filter/exclude_filesystem_usage
+            - filter/exclude_filesystem_inodes_usage
+            - filter/exclude_system_disk
+            - filter/exclude_system_paging
+            - filter/exclude_network
+            - attributes/exclude_system_paging
+            - resource
+            - k8sattributes
+            - batch
+          exporters: [otlphttp/newrelic, logging]
+        logs:
+          receivers: [otlp, filelog]
+          processors: [transform/truncate, resource, k8sattributes, batch]
+          exporters: [logging, otlphttp/newrelic]
```

Comment:
isn't there the risk of creating a loop having both a filelog receiver and the logging exporter? 🤔

Reply:
this was updated, to remove logging exporter since it is deprecated, but could you expand a bit more on this? the only reason I could think of this creating a loop was if the logging exporter exported logs to a log on the filesystem and then the filelog receiver would write those logs back out, but it was my understanding that the logging exporter only wrote logs out to the console, and it wasn't ever persisted/stored or written anywhere

Reply:
aren't we getting the logs of the pods as well with the log receiver?

Reply:
we are getting pod logs w/ the filelog receiver, yes. 

since we removed logging exporter, shouldn't have concerns with the loop :)